### PR TITLE
MC 2016 Wprime_VBF_WZ_inclu

### PR DESCRIPTION
**Pull request by IreneZoi**
I am adding script and example cards for Wprime_VBF_WZ_inclu for official 2016 production

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Compiled source #
```
Comment (perrozzi) [inline .gitignore]: this file is not relevant

**Comment by perrozzi** [comment]
we usually ask to provide one example mass point and then a script to produce the others to avoid overloading the repository, can you please update the PR?

**Comment by perrozzi** [comment]
hi, you apparently added files but didn't remove the others

**Comment by IreneZoi** [comment]
Sorry, I didn't understand what I should remove. I now have the script to produce all the mass points, the directory that is used by the script to produce the mass points and an example for a certain mass point. Is it this last one that I should remove? Thanks

**Comment by perrozzi** [comment]
I believe you have, in this PR, all the files for all the mass points and not just a representative datacard and the script to produce the other points (in fact there are 150 files). is this correct?

**Pull request merged**